CHANGELOG.md

```diff
@@ -1,5 +1,22 @@
 # Changelog
 
+## v1.6.2 - 2021-04-26
+
+### Bug fixes
+- CLI: Use the proper proxy command for `verdi calcjob gotocomputer` if configured as such [[#4761]](https://github.com/aiidateam/aiida-core/pull/4761)
+- Respect nested output namespaces in `Process.exposed_outputs` [[#4863]](https://github.com/aiidateam/aiida-core/pull/4863)
+- `NodeLinkManager` now properly regenerates original nested namespaces from the flat link labels stored in the database. This means one can now do `node.outputs.some.nested.output` instead of having to do `node.outputs.some__nested__output`. The same goes for `node.inputs` [[#4625]](https://github.com/aiidateam/aiida-core/pull/4625)
+- Fix `aiida.cmdline.utils.decorators.with_dbenv` always loading the database. Now it will only load the database if not already loaded, as intended [[#4865]](https://github.com/aiidateam/aiida-core/pull/4865)
+
+### Features
+- Add the `account` option to the `LsfScheduler` scheduler plugin [[#4832]](https://github.com/aiidateam/aiida-core/pull/4832)
+
+### Documentation
+- Update ssh proxycommand instructions [[#4839]](https://github.com/aiidateam/aiida-core/pull/4839)
+- Add the "How to extend workflows" section [[#4562]](https://github.com/aiidateam/aiida-core/pull/4562)
```

Comment:
explain this a bit further (extend workflows how?) and provide a link to where it is in the documentation

Reply:
@ramirezfranciscof you approved this PR. Can you provide this info? I think @mbercx is on holiday

Reply:
Extend workflows refers to including calls to other workflows inside of your workflow and the tools to adjust the interface (mostly the expose inputs/outputs). It refers to [this section](https://aiida.readthedocs.io/projects/aiida-core/en/latest/howto/write_workflows.html#extending-workflows). That works or do you want a more "ready to use" description? (in which case I would need parameters on the format 😬 )

Reply:
I added a comment on the relevant section, @chrisjsewell can you check it out to see if that is better?